src/components/tabs/tabs-test.stories.tsx

```diff
@@ -1,16 +1,16 @@
-import { Meta, Story, Canvas } from "@storybook/addon-docs";
+import React from "react";
+import { Tabs, Tab, TabsProps } from ".";
 
-import { Tabs, Tab } from "./tabs.component";
-
-<Meta
-  title="Tabs/Test"
-  parameters={{
+export default {
+  title: "Tabs/Test",
+  includeStories: "Default",
```

Comment:
Is this line necessary? It should be included automatically

Reply:
It seems that it indeed isn't necessary - I'll remove it